Core/src/main/java/com/plotsquared/core/listener/PlotListener.java

```diff
@@ -200,10 +200,11 @@ public boolean plotEntry(final PlotPlayer<?> player, final Plot plot) {
                             Caption caption = TranslatableCaption.of("notification.notify_enter");
                             Template playerTemplate = Template.of("player", player.getName());
                             Template plotTemplate = Template.of("plot", plot.getId().toString());
+                            Template worldTemplate = Template.of("world", plot.getWorldName());
                             if (!Settings.Chat.NOTIFICATION_AS_ACTIONBAR) {
-                                owner.sendMessage(caption, playerTemplate, plotTemplate);
+                                owner.sendMessage(caption, playerTemplate, plotTemplate, worldTemplate);
                             } else {
-                                owner.sendActionBar(caption, playerTemplate, plotTemplate);
+                                owner.sendActionBar(caption, playerTemplate, plotTemplate, worldTemplate);
```

Comment:
That could be moved into a private constructor, we're basically reusing the entire block at #plotExit below.